plugins/amazonq/codewhisperer/jetbrains-community/resources/META-INF/plugin-codewhisperer.xml

```diff
@@ -81,9 +81,31 @@
         </group>
 
         <action class="software.aws.toolkits.jetbrains.services.codewhisperer.actions.CodeWhispererRecommendationAction"
-                text="Show Code Suggestions">
+                text="Invoke Amazon Q Inline Suggestions">
             <keyboard-shortcut keymap="$default" first-keystroke="alt C"/>
         </action>
+        <action id="codewhisperer.inline.navigate.previous"
+                class="software.aws.toolkits.jetbrains.services.codewhisperer.actions.CodeWhispererNavigatePrevAction"
+                text="Navigate to Previous Inline Suggestion" description="Navigate to previous inline suggestion">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt OPEN_BRACKET"/>
+        </action>
+        <action id="codewhisperer.inline.navigate.next"
+                class="software.aws.toolkits.jetbrains.services.codewhisperer.actions.CodeWhispererNavigateNextAction"
+                text="Navigate to Next Inline Suggestion" description="Navigate to next inline suggestion">
+            <keyboard-shortcut keymap="$default" first-keystroke="alt CLOSE_BRACKET"/>
+        </action>
+        <action id="codewhisperer.inline.accept"
+                class="software.aws.toolkits.jetbrains.services.codewhisperer.actions.CodeWhispererAcceptAction"
+                text="Accept the Current Inline Suggestion" description="Accept the current inline suggestions">
+            <keyboard-shortcut keymap="$default" first-keystroke="TAB"/>
```

Comment:
i'm a bit worried about the interaction between this and the IDE

Reply:
The logic will be what the promoter does -- when the CW is showing:
` CodeWhispererInvocationStatus.getInstance().isDisplaySessionActive()`
it will be promoted to CW accept.